Dockerfile

```
@@ -1,21 +1,32 @@
+# We need the non-slim image for the apify-cli installation
 FROM nikolaik/python-nodejs:python3.7-nodejs16 as build
-WORKDIR /wheels
-COPY requirements.txt /opt/sherlock/
-RUN pip3 wheel -r /opt/sherlock/requirements.txt
 
+# Install the Apify CLI globally, to /usr/lib/node_modules
+RUN npm install -g apify-cli@0.7.1-beta.4
```

```suggestion
RUN npm install -g apify-cli
```

I published it as latest.